packages/sitecore-jss-nextjs/src/editing/editing-healthz-middleware.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiResponse, NextApiRequest } from 'next';
+
+/**
+ * Middleware / handler for use in the healthz Next.js API route (e.g. '/api/healthz')
+ * which is used by XMCloud container editing host.
```

Comment:
Update comment to be more generic
```suggestion
 * Middleware / handler for use in healthcheck Next.js API route (e.g. '/api/healthz').
```